buildchain/buildchain/salt_tree.py

```diff
@@ -113,6 +113,7 @@ def task__deploy_salt_tree() -> Iterator[types.TaskDict]:
     Path('salt/metalk8s/bootstrap/preflight.sls'),
     Path('salt/metalk8s/bootstrap/registry.sls'),
     Path('salt/metalk8s/bootstrap/salt_master.sls'),
+    Path('salt/metalk8s/bootstrap/services.sls'),
```

Comment:
So, I was wondering, do we need such a formula? If we follow the structure defined in https://github.com/scality/metalk8s/pull/967, I guess we could have a `deployed.sls` in all "services", i.e. `metalk8s.registry`, `metalk8s.repo` and `metalk8s.salt.master`. All these will be called as soon as we have an API server ready.

Reply:
@TeddyAndrieux any thought on this? Is there anything I can change in my PR to facilitate https://github.com/scality/metalk8s/pull/967 ? FYI: for now I can move the service object deployment in `salt/metalk8s/salt/master/deployed.sls` because when the state deployed.sls is applied, the API server is not up yet. Does it make sense for you if this change is managed in https://github.com/scality/metalk8s/pull/967 ?

Reply:
As for the moment we already have `deployed.sls` for salt master and repo (which is not real "deployed" as we want to use in our salt states)
I think you should keep the naming `service_deployed.sls` for the moment. I will rename them in my PR